src/kv/MemDB.h

```diff
@@ -122,8 +122,16 @@ class MemDB : public KeyValueDB
   int _init(bool format);
 
   int do_open(ostream &out, bool create);
+
   int open(ostream &out) override { return do_open(out, false); }
+  int open(ostream &out, const std::vector<ColumnFamily>& cfs) override {
+    return KeyValueDB::open(out, cfs);
+  }
+
```

Comment:
do we need these implementations? It will fall back to parent anyways, if it not a pure virtual function. We have the override only for the supported implementations.

Reply:
yes, I will try to remove it, thanks.